README.rst

```diff
@@ -184,6 +184,7 @@ The following options can be configured on the server:
     datadir                                   ./data                                                                                                                                                                                                                                                                                                               Directory where the node stores its files.
     internalratelimiter                       true                                                                                                                                                                                                                                                                                                                 When set, expensive internal calls are rate-limited to protect the network. Always enabled in strict mode.
     loggerformat                              text                                                                                                                                                                                                                                                                                                                 Log format (text, json)
+    methods                                   [nuts]                                                                                                                                                                                                                                                                                                               List of enabled DID methods
```

Comment:
Mainly a future concern;
- There could be DID methods that you don't want to enable by default, e.g. experimental ones
- Future DID methods only applicable when used in specific use cases
- Maybe you want to disable `did:nuts` :-p

But since it's many future "if's", we can refrain from making it configurable for now?

Reply:
I ask because it has quite some impact on the dependencies (also on startup). I think it can be removed for now.